devicetypes/smartthings/zwave-metering-dimmer.src/zwave-metering-dimmer.groovy

```diff
@@ -295,6 +295,25 @@ def zwaveEvent(physicalgraph.zwave.commands.crc16encapv1.Crc16Encap cmd) {
 	}
 }
 
+def zwaveEvent(physicalgraph.zwave.commands.multichannelv3.MultiChannelCmdEncap cmd) {
+	if (cmd.commandClass == 0x6C && cmd.parameter.size >= 4) { // Supervision encapsulated Message
```

Comment:
@kianooshST is this still necessary? I thought this was a temp fix until it was handled in hubcore?

Reply:
@greens @kianooshST Does hubcore do anything special with unwrapping the packet that this DTH isn't accounting for? (I believe that this DTH is supposed to be local, but I guess not?)

Reply:
@kianooshkarami ping

I am probably going to remove this block of code (there is another line that I think I left out of this PR -- which would explain why SRPOL still had this not work).

Reply:
Yes sorry. Please go ahead and remove this bit of code.

Reply:
@kianooshkarami Ok, I will have to see if the device I have still doesn't work after removing this. It didn't a month ago.

Reply:
@kianooshkarami Ok! I think I'll keep it, unless @greens or anyone else have any strong opinions.